package.json

```diff
@@ -18,6 +18,7 @@
     "prettier:diff": "prettier --list-different \"**/*.{scss,css,js,md}\"",
     "semantic-release": "semantic-release",
     "start": "yarn storybook",
+    "start:experimental": "cross-env CARBON_USE_EXPERIMENTAL_FEATURES=true CARBON_REACT_STORYBOOK_USE_EXTERNAL_CSS=true CARBON_REACT_STORYBOOK_USE_STYLE_SOURCEMAP=true yarn start",
```

Comment:
If this is (solely) for running Storybook with experimental mode, the only environment variable change needed is `CARBON_USE_EXPERIMENTAL_FEATURES=true`.